website/src/pages/Status/Status.jsx

```diff
@@ -15,12 +15,133 @@ limitations under the License.
 */
 
 import React from 'react';
+import { useState, useEffect } from 'react';
+
+import './status.css'
+
+import PreviousExe from '../../components/PreviousExecutions/PreviousExe';
+import ExeQueue from '../../components/ExecutionQueue/ExeQueue';
+import RingLoader from "react-spinners/RingLoader";
```

Comment:
I feel like this import should be move above, among the other two external imports.